test failure with python3 - test_mime_types (test.test.MagicTest) ... FAIL

```python
test_mime_types (test.test.MagicTest) ... FAIL

======================================================================
FAIL: test_keep_going (test.test.MagicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/tmpfs/python-magic/test/test.py", line 79, in test_keep_going
    self.assertEqual(m.from_file(filename), 'image/jpeg'.encode('utf-8'))
AssertionError: b'image/jpeg\\012- application/octet-stream' != b'image/jpeg'

======================================================================
FAIL: test_mime_types (test.test.MagicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/tmpfs/python-magic/test/test.py", line 36, in test_mime_types
    b'\xce\xbb': 'text/plain',
  File "/mnt/tmpfs/python-magic/test/test.py", line 23, in assert_values
    self.assertEqual(value, expected_value_bytes)
AssertionError: b'application/x-gzip' != b'application/gzip'

----------------------------------------------------------------------
Ran 5 tests in 0.019s

FAILED (failures=2)
```

--- a/test/test.py
+++ b/test/test.py
@@ -54,7 +54,7 @@ class MagicTest(unittest.TestCase):
             self.assert_values(m, {
                 'magic._pyc_': 'application/octet-stream',
                 'test.pdf': 'application/pdf',
-                'test.gz': 'application/gzip',
+                'test.gz': ('application/gzip', 'application/x-gzip'),
                 'text.txt': 'text/plain',
                 b'\xce\xbb'.decode('utf-8'): 'text/plain',
                 b'\xce\xbb': 'text/plain',

